pkg/apiserver/apiserver_test.go

```diff
@@ -1234,8 +1242,9 @@ func TestMetadata(t *testing.T) {
 		}
 	}
 	if matches["text/plain,application/json,application/yaml,application/vnd.kubernetes.protobuf"] == 0 ||
+		matches["application/json,application/json;stream=watch,application/vnd.kubernetes.protobuf,application/vnd.kubernetes.protobuf;stream=watch"] == 0 ||
```

Comment:
Tempted to refactor this but it's kind of ugly in general to calculate this
any other way.

On Mon, Apr 25, 2016 at 11:17 AM, David Eads <notifications@github.com>
wrote:

> In pkg/apiserver/apiserver_test.go
> <https://github.com/kubernetes/kubernetes/pull/24710#discussion_r60931185>
> :
>
> > @@ -1234,8 +1242,9 @@ func TestMetadata(t *testing.T) {
> >  		}
> >  	}
> >  	if matches["text/plain,application/json,application/yaml,application/vnd.kubernetes.protobuf"] == 0 ||
> > +		matches["application/json,application/json;stream=watch,application/vnd.kubernetes.protobuf,application/vnd.kubernetes.protobuf;stream=watch"] == 0 ||
>
> yeouch.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/24710/files/e4a1f3c4b34ba5ef0c8f08009e202d0b048498ff#r60931185>
>